albo.py

```diff
@@ -97,5 +104,7 @@ def send_telegram_attachments(pdf_links: List[str]) -> None:
       send_telegram_message(error_string, dev)
   #print(message)
 
-with open("last_id.txt", "w+") as f:
-    f.write(str(new_id))
+with open("last_id.txt", "w+") as f_id, open("cached_announcements.txt", "w+") as f_cached:
+    f_id.write(str(new_id))
+    for cached_id in cached_announcements:
+      f_cached.write(str(cached_id) +  "\n")
```

Comment:
from your point of view, does it has sense to move the second file inside the first `with open()`?
```python
with open("last_id.txt", "w+") as f_id:
    f_id.write(str(new_id))
   with open("cached_announcements.txt", "w+") as f_cached:
        for cached_id in cached_announcements:
          f_cached.write(str(cached_id) +  "\n")
```

Reply:
or.. make it separate?

```python
with open("last_id.txt", "w+") as f_id:
    f_id.write(str(new_id))

with open("cached_announcements.txt", "w+") as f_cached:
    for cached_id in cached_announcements:
        f_cached.write(str(cached_id) +  "\n")
```